internal/core/converter/tensorflow/tensorflow.go

```diff
@@ -29,6 +29,15 @@ type TF interface {
 	GetVector(inputs ...string) ([]float64, error)
 	GetValue(inputs ...string) (interface{}, error)
 	GetValues(inputs ...string) (values []interface{}, err error)
+	Closer
+}
+
+type session interface {
+	Run(feeds map[tf.Output]*tf.Tensor, fetches []tf.Output, operations []*Operation) ([]*tf.Tensor, error)
+	Closer
+}
+
+type Closer interface {
```

Comment:
@datelier 
Could you please add commen for `Closer`?

Reply:
I think we can use the standard `Closer` interface, what do you think?
https://golang.org/pkg/io/#Closer

Reply:
@datelier @hlts2 how about this suggestion?

Reply:
thank you for your comment. I understand your opinion. But I think it's better not to use `io` package. Because if go developer modify `Closer` interface of `io` package (e.g. add parameter ...etc), this package will not work well.

Reply:
I think if we think this way we should not use all go standard package....

Both are correct but for me it is not a concern because `Closer` interface is generalized enough to use...
@kpango what do you think?